src/util.js

```diff
@@ -1,28 +1,5 @@
 import { EMPTY_ARR } from './constants';
 
 export const isArray = Array.isArray;
-
-/**
- * Assign properties from `props` to `obj`
- * @template O, P The obj and props types
- * @param {O} obj The object to copy properties to
- * @param {P} props The object to copy properties from
- * @returns {O & P}
- */
-export function assign(obj, props) {
```

Comment:
We were using this because it's faster, probably the reason the benchmarks show a drop.